lib/naughty/list/notifications.lua

```diff
@@ -367,8 +376,6 @@ function module.filter.most_recent(n, count)
     return false
 end
 
---@DOC_widget_COMMON@
-
 --@DOC_object_COMMON@
```

Comment:
Right, I think we can remove those `.ldoc` now. This can be done later in another PR. I only  removed the ones with caused some warnings in the linting code.